### ISSUE TITLE
Shatterspleen description doesn't mention the 5% critical chance it gives

### ISSUE THREAD

**Issue by InQuognito** [xoxfaby/BetterUI#101]
Title, I didn't use the format because it's a very small thing. Latest version as of posting.

**Comment by xoxfaby** [comment]
For now, BetterUI doesn't actual change any item descriptions, though that is something I'm considering. And since it doesn't stack, I decided not to add it to ItemStats. But I am open to hearing opinions on what BetterUI should do about things like this

**Comment by InQuognito** [comment]
Yes it does? Harvester's Scythe says "Gain 5% critical strike chance" but not in Vanilla.

**Comment by InQuognito** [comment]
Doesn't it change every item description? Or am I misunderstanding you comment?

**Comment by xoxfaby** [comment]
It doesn't, it changes the item descriptions to the logbook descriptions ( still vanilla ) and attaches the ItemStats to the end.

**Comment by InQuognito** [comment]
ohh okay that makes sense! well then yeah, I think changing some descriptions would be a good idea
